pkg/cmd/server/start/start_master.go

```diff
@@ -242,10 +242,10 @@ func (o MasterOptions) RunMaster() error {
 			return err
 		}
 
-		if err := os.MkdirAll(path.Dir(o.MasterArgs.GetConfigFileToWrite()), os.FileMode(0755)); err != nil {
```

Comment:
Ok, this is fine then, although it's probably going to break a lot of
instructions (anyone who calls sudo and then uses the config file).  Is
there a docs PR to link to this?

On Fri, Apr 8, 2016 at 2:17 PM, Scott Dodson <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_master.go
> <https://github.com/openshift/origin/pull/8408#discussion_r59067620>:
>
> > @@ -242,10 +242,10 @@ func (o MasterOptions) RunMaster() error {
> >  			return err
> >  		}
> >
> > -		if err := os.MkdirAll(path.Dir(o.MasterArgs.GetConfigFileToWrite()), os.FileMode(0755)); err != nil {
>
> @smarterclayton <https://github.com/smarterclayton> It looks like there
> are several examples of this being somewhat of a norm. /etc/grub.d and
> /etc/libvirt which are 0700 but files under them are more permissive.
> Apparently the reason that /etc/pki is open wider is that it's used by a
> wide variety of tools. Unless we're saying that something needs access to
> the ca.crt in /etc/origin/master I think we should tighten them. I don't
> think that we need access to that because it should be in the kubeconfig as
> well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8408/files/db0365c1aff40756cac21e4ade76c7eb798c4b91#r59067620>
>